docs/v0.3/sdk/go/fssrz-index.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: /fssrz/sdk-go
```

Comment:
This is the URL for the landing page of the Go SDK tech preview docs. Please lmk if it needs to be changed.

Reply:
Looks good to me. WDYT @shykes?